src/DataStructures/Tensor/EagerMath/OrthonormalOneform.hpp

```diff
@@ -0,0 +1,74 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+// @{
+/*!
+ * \ingroup TensorGroup
+ *
+ * \brief Compute a spatial one-form orthonormal to the given unit form
+ *
+ * Given a unit spatial one-form \f$s_i\f$, compute a new form \f$t_i\f$
+ * which is orthonormal to \f$s_i\f$, in the sense that
+ * \f$\gamma^{ij}s_i t_j = 0\f$, for the given inverse spatial metric
+ * \f$\gamma^{ij}\f$.
+ *
+ * \details The new form is obtained via Gram-Schmidt process, starting
+ * from a form whose components are \f$t_i = \delta_i^I\f$, where \f$I\f$ is
+ * the index of the smallest absolute value of the components of \f$s_i\f$.
```

Comment:
1) I would find this clearer phrased as "... the index of the component of s_i with the smallest absolute value."

2) For my understanding: if `s_i` varies spatially (i.e. `DataType = DataVector`) in such a way that the smallest component changes between grid points, should I expect the resulting `t_i` to be discontinuous? If that's true, perhaps this should be noted?

Reply:
1. Thanks, your way sounds much clearer.
2. Yes. More generally, if s_i changes from point to point, then t_i is expected to vary as well. I thought this was evident (e.g radial and tangential basis vectors in polar coordinates change from point to point) but if you find it necessary, I can add an extra comment.

Reply:
I agree it is evident that if s_i varies spatially, then t_i should also vary spatially. But I think it may be nice to add a note that even for smoothly-varying s_i, the output t_i could be discontinuously-varying. Up to you, though 🙂

Reply:
I was trying to come up with some example to illustrate this point so I could add it to the dox, but I couldn't find any. I'll update the dox in the future if I come up with something.